freelist_test.go

```diff
@@ -160,7 +161,7 @@ func TestFreelist_releaseRange(t *testing.T) {
 		}
 
 		for _, p := range c.pagesIn {
-			f.free(p.freeTxn, &page{id: p.id})
+			f.free(p.freeTxn, &page{id: p.id, overflow: uint32(p.n - 1)})
```

Comment:
why this needs to be changed?

Reply:
because the old test is wrong, the page 9 and 10 can be both freed, because it is in release range {151, 300}.